.ci/lib.sh

```diff
@@ -464,6 +464,23 @@ gen_clean_arch() {
 	# make sure cargo and rustup home dir are removed
 	info "remove cargo and rustup home dir"
 	sudo rm -rf ~/.cargo ~/.rustup
+
+	# make sure coco artifacts are removed
+	info "remove coco artifacts"
+	sudo rm -rf /opt/confidential-containers
+
+	# make sure to clean up binaries
+	unlink /opt/confidential-containers/share/ovmf
```

Comment:
... and then here you're trying to unlink a dir that was just removed in a few lines above.

As this pattern repeats in all the removals, so I'd suggest to move the `sudo rm -rf /opt/confidential-containers` to be the last part of the cleanup.